group-coordinator/src/main/java/org/apache/kafka/coordinator/group/share/ShareCoordinatorShard.java

```diff
@@ -251,20 +251,18 @@ public CoordinatorResult<WriteShareGroupStateResponseData, Record> writeState(Re
     WriteShareGroupStateResponseData responseData = new WriteShareGroupStateResponseData();
     for (Record record : recordList) {  // should be single record
       if (record.key().message() instanceof ShareSnapshotKey && record.value().message() instanceof ShareSnapshotValue) {
-        ShareSnapshotKey newKey = (ShareSnapshotKey) record.key().message();
-        ShareSnapshotValue newValue = (ShareSnapshotValue) record.value().message();
+        ShareSnapshotKey recordKeyKey = (ShareSnapshotKey) record.key().message();
```

Comment:
`recordKeyKey`??? I know it's trivial but please can we have a less comical name :)

Reply:
Thanks for the review, sorry about this 😅 The changes are there in the latest commit